variants/STM32MP157_DK/README.md

```diff
@@ -13,49 +13,49 @@ Because every OS may have different software configurations (especially Device T
 * [STM32 MPU OpenSTLinux Distribution]
 * [Balena OS]
 
-Note that the first two OSes should select `stm32mp157c-dk2-m4-examples-sdcard` boot mode (or device tree configuration) on boot time. `stm32mp157c-dk2-sdcard` boot mode is also known to work but it is not guaranteed. See [ST Wiki page on boot mode] for more detail. If you are using it for your own OS, see the [Linux Device Tree considerations](#Linux-Device-Tree-considerations) section.
+__Note that the first two OSes should select `stm32mp157c-dk2-m4-examples-sdcard` boot mode (or device tree configuration) on boot time.__ `stm32mp157c-dk2-sdcard` boot mode is also known to work but it is not guaranteed. See [ST Wiki page on boot mode] for more detail. If you are using it for your own OS, see the [Linux Device Tree considerations](#Linux-Device-Tree-considerations) section.
 
 ## How to use
 
 After Verify and Upload, you will see a message similar to the following in Arduino IDE:
 
-    <Arduino build output path>/run_arduino.sh generated successfully.
+    <Arduino build output path>/run_arduino_<sketch name>.sh generated successfully.
     This file should be uploaded manually by SCP, SFTP, Kermit, or etc.
-    Then run "sh ./run_arduino.sh start" command in the board's console.
+    Then run "sh ./run_arduino_<sketch name>.sh start" command in the board's console.
     For detailed instructions, please visit:
         https://github.com/stm32duino/Arduino_Core_STM32/tree/master/variants/STM32MP157_DK/README.md
 
-In this example, the user **must** upload `<Arduino build output path>/run_arduino.sh` file manually. Uploading instruction is described later in the [Uploading](#Uploading) section. 
+In this example, the user **must** upload `<Arduino build output path>/run_arduino_<sketch name>.sh` file manually. Uploading instruction is described later in the [Uploading](#Uploading) section. 
```

Comment:
In my opinion, the new name of `<Arduino build output path>/run_arduino_<your project name>.sh` reduces readability of the document and users may take more time to notice what `<Arduino build output path>/run_arduino_<your project name>.sh` is. So I think adding example samples of `<Arduino build output path>/run_arduino_<your project name>.sh` will help the users. What about adding the example after this paragraph?
```markdown
`<Arduino build output path>/run_arduino_<your project name>.sh` looks like this:

* `C:/Users/kbumsik/AppData/Local/Temp/arduino_build_668148/run_arduino.Blink.ino.sh` (For Windows)
* `/tmp/arduino_build_668148/run_arduino_Blink.ino.sh` (For Linux/macOS)
```